swift3/controllers/multi_upload.py

```diff
@@ -602,6 +602,10 @@ def POST(self, req):
         container = req.container_name + MULTIUPLOAD_SUFFIX
         resp = req.get_response(self.app, 'GET', container, '', query=query)
         objinfo = json.loads(resp.body)
+
+        # pylint: disable-msg=E1103
```

Comment:
Prefer using the "readable" message name. "no-member" is obvious whereas "E1101" is not.